docs/OncoTree-Version-Converter.md

```diff
@@ -0,0 +1,80 @@
+
+## [Running the tool](#running-the-tool)
+
+The OncoTree Version Converter tool can be run with the following command:
+
+
+```
+python <path/to/scripts/convert_oncotree_version.py> --input-file <path/to/input/file> --output-file <path/to/output/file> --source-version <source_oncotree_version> --target-version <target_oncotree_version> --oncotree-url <oncotree_instance_url>
+```
+
+**Options**
+- `-i | --input-file`: This is the input clinical file path. It must contain `ONCOTREE_CODE` in the file header and it must contain OncoTree codes corresponding to the `<source_oncotree_version>`. Read more about the cBioPortal clinical file format [here](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#clinical-data).
+- `-o | --output-file`: This is the path to the output clinical file that will be generated. It will contain mapped OncoTree codes from `<source_oncotree_version>`-to-`<target_oncotree_version>` and a new column called `ONCOTREE_SUGGESTIONS` containing suggestions for OncoTree codes if a direct mapping was not available.
+- `-s | --source-version`: This is the source OncoTree version. The OncoTree codes in the input file must correspond to this version.
+- `-t | --target-version`: This is the target OncoTree version that the script will attempt to map the input file OncoTree codes to.
+- `-u | --oncotree-url`: This is the base OncoTree URL to use. If not supplied, the default will be used (http://oncotree.mskcc.org/api/)
+
+The list of OncoTree versions available are viewable [here](http://oncotree.mskcc.org/api/versions) on the dropdown menu [OncoTree home page](http://oncotree.mskcc.org/#/home).
+
+**Required Python modules:**
+
+- [requests](http://docs.python-requests.org/en/v2.7.0/user/install/)
+
+## [OncoTree History Modeling](#oncotree-history-modeling)
+
+### Background
+
+The OncoTree has gone through several iterations since its conception to improve the classification of detailed cancer type information. As such, some nodes on the OncoTree possess a complex revision history whereas others have a much more simple revision history, such as a simple renaming of the detailed cancer type name.
+
+The complexity of the revision history for some OncoTree nodes created a need for a tool that facilitates the mapping of OncoTree codes between OncoTree release versions.
+
+### Schema updates and new properties explained
+
+Three new properties were introduced as a result of the OncoTree history modeling. Each node will now contain the following new properties and will be discussed in further detail below.
```

Comment:
I actually don't think we need to discuss the JSON format.  I'd rather have examples of each type of mapping case (like the sections in our status html file) that describes why the tool does what it does - diagrams ala @sheridancbio would be a nice to have here.